src/Kamino.ts

```diff
@@ -2130,6 +2130,12 @@ export class Kamino {
             name: getTokenNameFromCollateralInfo(collateralInfo),
           };
         }
+      } else {
+        const jupPrice = JupService.getDollarPrice(collateralInfo.mint);
```

Comment:
Please wrap this in a try-catch and skip if there is any kind of error (you can log the error so we know sth happened to a specific mint). We don't want this method to fail because the Jup API is down or something.